java/src/test/java/ai/onnxruntime/InferenceTest.java

```diff
@@ -55,10 +55,16 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
 import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+import org.junit.jupiter.api.condition.OS;
 
 /** Tests for the onnx-runtime Java interface. */
 public class InferenceTest {
+  private static final String qnnEpBackendPathEnvironmentVariableName =
+      "ORT_JAVA_TEST_QNN_EP_BACKEND_PATH";
```

Comment:
getting the path through an environment variable feels clunky. open to suggestions for a better way to do this.

Reply:
For C++ unit tests, the build process copies the backend .so/.dll files to the same directory as the test executable. So, our tests typically can just set the backend path to `libQnnHtp.so`. Is there a way to do something similar here?

Reply:
it seems to work on Windows if I copy `${QNN_LIB_FILES}` to `${JAVA_NATIVE_TEST_DIR}` and then use a backend_path of "QnnCpu.dll".

https://github.com/microsoft/onnxruntime/blob/3846f84218a77b0071b5ed992832a328870bc84f/cmake/onnxruntime_unittests.cmake#L1589

I'm not sure if this is the right place to copy the libs though.

Reply:
That's probably fine, they won't end up packaged in anything there so it shouldn't cause trouble.

Reply:
thanks for the input. updated to remove the environment variable and copy the QNN libraries and then get the QNN library path with TestHelpers.getResourcePath().